config.js

```diff
@@ -191,7 +191,6 @@ config.search = {
                     },
                     'html_content': {
```

Comment:
Now that we'll be indexing plain text, do we still need the html_content analyzer?